tbx/propositions/migrations/0016_delete_subservicepagetosubpropositionpagemigration.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 3.2.20 on 2023-11-02 11:11
+
+from django.db import migrations
+
+
+def delete_pages(apps, schema_editor):
+    SubServicePageToSubPropositionPageMigration = apps.get_model(
+        "propositions", "SubServicePageToSubPropositionPageMigration"
+    )
+    SubServicePageToSubPropositionPageMigration.objects.all().delete()
+
+
+def nooperation(apps, schema_editor):
+    pass
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("propositions", "0015_add_optional_link_to_testimonials"),
+    ]
+
+    state_operations = [
+        migrations.DeleteModel(
+            name="SubServicePageToSubPropositionPageMigration",
+        ),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            state_operations=[],
+            database_operations=[
+                migrations.RunPython(delete_pages, nooperation),
+            ],
+        ),
```

Comment:
This doesn't need to be in the `SeparateDatabaseAndState`.

```suggestion
        migrations.RunPython(delete_pages, migrations.RunPython.noop),
```

Also a trick I learnt is to use `migrations.RunPython.noop` for the reverse_code. This way you don't need to define an empty nooperation method above.